src/main/org/nlogo/window/ButtonWidget.scala

```diff
@@ -62,8 +61,8 @@ object ButtonWidget {
   // encapsulates what used to be a bunch of 4 way if statements.
   // ButtonWidget now has a single ButtonType object that handles all this logic for it.
   case class ButtonType(name: String, agentClass:Class[_ <: Agent],
-                        img:Option[ImageIcon], darkImg:Option[ImageIcon]){
-    def img(dark:Boolean): Option[ImageIcon] = if(dark) darkImg else img
+                        img:Option[ImageIcon], inverseImg:Option[ImageIcon]){
```

Comment:
Instead of renaming `darkImg` to `inverseImg`, let's rename it to something like `activeImg`. If you would like, you can also rename `img` to `inactiveImg`.